backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/search/predicate/impl/AbstractElasticsearchPredicate.java

```diff
@@ -53,6 +77,10 @@ public JsonObject toJsonQuery(PredicateRequestContext context) {
 	protected abstract JsonObject doToJsonQuery(PredicateRequestContext context,
 			JsonObject outerObject, JsonObject innerObject);
 
+	protected JsonElement doToJsonKnn(PredicateRequestContext context) {
+		return null;
+	}
```

Comment:
This is probably worth some comments... ?

Reply:
not anymore 😄 (see commit with message `HSEARCH-4950 Hide knn clauses in a predicate request context`)